libjolie/src/main/java/jolie/lang/KeywordClass.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright (C) 2021 Fabrizio Montesi <famontesi@gmail.com>
+ * Copyright (C) 2021 Vicki Mixen <vicki@mixen.dk>
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
+ * MA 02110-1301  USA
+ */
+
+package jolie.lang;
+
+import java.util.Arrays;
+import java.util.List;
+
+public class KeywordClass {
+	/**
+	 * 
+	 * @return List of keywords that can be used inside the inputPort scope
+	 */
+	public static List< String > inputPortKeywords() {
+		List< String > keywords = Arrays.asList( "location", "protocol", "interfaces", "aggregates", "redirects",
+			"RequestResponse", "OneWay" );
+		return keywords;
```

Comment:
Consider instead making a private static List inside of this class and return that.